src/components/DataSubmissions/DataSubmissionSummary.tsx

```diff
@@ -97,41 +87,39 @@ const StyledSectionDivider = styled(Divider)(() => ({
     display: "flex",
     alignSelf: "flex-start",
     width: "2px",
-    height: "159px",
+    height: "157px",
     background: "#6CACDA",
-    marginLeft: "44px",
-    marginTop: "9px",
+    marginLeft: "48px",
+    marginRight: "81px",
   },
 }));
 
-const StyledSubmitterName = styled(StyledValue)(() => ({
-  whiteSpace: "nowrap",
-  overflow: "hidden",
-  textOverflow: "ellipsis",
-  maxWidth: "100%",
-  lineHeight: "19.6px",
-  flexShrink: 1,
-}));
-
 const StyledConciergeName = styled(StyledValue)(() => ({
   maxWidth: "100%",
   lineHeight: "19.6px",
   flexShrink: 1,
 }));
 
-const StyledTooltipSubmitterName = styled(StyledValue)(() => ({
-  color: "#595959",
-  fontFamily: "'Nunito'",
+const StyledCollaboratorsButton = styled(Button)({
+  justifyContent: "flex-start",
+  padding: 0,
+  margin: 0,
+  color: "#0B7F99",
+  fontFamily: "'Nunito', 'Rubik', sans-serif",
   fontSize: "16px",
   fontStyle: "normal",
```

Comment:
I forget that it is the default, I will try to avoid this issue in the future. Updated in [326be8f](https://github.com/CBIIT/crdc-datahub-ui/pull/497/commits/326be8f9c5077b1ed5e0170e323c7ef692c35e3c). Thanks!

Reply:
No worries, I also forget to check the default styling usually. Thanks